openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -380,6 +381,19 @@ def has_ended(self):
         """
         return course_metadata_utils.has_course_ended(self.end)
 
+    def has_marketing_url(self):
+        """
+        Returns whether the course has marketing url.
+        """
+        return settings.FEATURES.get('ENABLE_MKTG_SITE') and self.marketing_url
```

Comment:
@Qubad786 This will return the actual marketing url. From the method name the expectation is that a bool will be returned. As per discussion with @muzaffaryousaf we should change the method name and update the docstring and keep the implementation as it is.